package.json

```diff
@@ -39,7 +39,7 @@
     "test:watch": "jest --watch --verbose false --silent false",
     "test:node:playwright": "cd test/integration/node-browser && npm install && cp ../../../dist/umd/supabase.js . && npm run test",
     "test:bun": "cd test/integration/bun && bun install && bun test",
-    "test:types": "run-s build:module && tsd --files test/types/*.test-d.ts && jsr publish --dry-run",
+    "test:types": "run-s build:module && tstyche && jsr publish --dry-run",
```

Comment:
What if the `jsr` part would be move to a separate `test:jsr` script? This way `test:types` could be passed `--target '>=4.7'` in CI.